.github/workflows/dotnetcore-ubuntu.yml

```diff
@@ -47,7 +47,6 @@ jobs:
         cd tests/compose/standalone
         docker-compose ps
         docker-compose logs init
-        docker-compose logs initTls
```

Comment:
We'll need a separate cluster, since in one cluster we'll need maxMessageSize to be default and in another one increased.